mvc/views/tag-helpers/authoring.rst

```diff
@@ -1,8 +1,376 @@
-.. include:: /../common/stub-topic.txt
-
-|stub-icon| Authoring Tag Helpers
+Authoring Tag Helpers
 ===========================================
 
-.. include:: /../common/stub-notice.txt
+By `Rick Anderson`_
+
+    - `Getting started with Tag Helpers`_
```

Comment:
Switched to lc tag helpers for entire doc.

From: N. Taylor Mullen [mailto:notifications@github.com]
Sent: Wednesday, September 2, 2015 2:39 PM
To: aspnet/Docs <Docs@noreply.github.com>
Cc: Rick Anderson <Rick.Anderson@microsoft.com>
Subject: Re: [Docs] new TH authoring article (#419)


In mvc/views/tag-helpers/authoring.rst<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2faspnet%2fDocs%2fpull%2f419%23discussion_r38588089&data=01%7c01%7cRick.Anderson%40microsoft.com%7c4849c5075c744bb8eda908d2b3def7ae%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=6vDUT9ehIwxYVYdagRIwKiHIhQ55MtVUwy5nAHaqCsM%3d>:

>  ===========================================

>

> -.. include:: /../common/stub-notice.txt

> +By `Rick Anderson`_

> +

> +    - `Getting started with Tag Helpers`_

Should stay consistent with case: Tag Helpers or tag helpers

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2faspnet%2fDocs%2fpull%2f419%2ffiles%23r38588089&data=01%7c01%7cRick.Anderson%40microsoft.com%7c4849c5075c744bb8eda908d2b3def7ae%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=518tCV6IEUjnJCWji6lJv43dtlVK%2bQT%2fpMhuCHZIhFs%3d>.